.github/workflows/create_rc_pr.yml

```diff
@@ -1,7 +1,9 @@
 name: Create RC PR
 
 on:
-  workflow_dispatch:
+  schedule:
+    - cron: '0 14 * * 1,3,5' # Run on Monday, Wednesday, and Friday at 16:00 UTC
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Isn't it at 14:00 UTC ?

Reply:
yes I updated the code but not the comment...